config_files/kworkflow-network-default.xml

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
How about set static IP in each VM that Kw manages?
1. Mount vm disk (vm_mount)
2. Detect the distribution
3. Change the dhcp.conf by one of the IPs that we manage
4. Umount the vm disk (vm_umount)

Remember, we have to manage the IPs. Probably, we want to do it in a different file.